components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/builder/LogoutRequestBuilder.java

```diff
@@ -262,4 +252,20 @@ private static XMLObject buildXMLObject(QName objectQName) throws Exception {
         return builder.buildObject(objectQName.getNamespaceURI(), objectQName.getLocalPart(),
                 objectQName.getPrefix());
     }
+
+    /**
+     * Build the NameID object
+     *
+     * @return NameID object
+     */
+    private static NameID buildNameID(String nameIdFormat, String subject) {
+        NameID nameIdObj = new NameIDBuilder().buildObject();
+        if (nameIdFormat != null && !nameIdFormat.isEmpty()) {
```

Comment:
You can use commons StringUtils.isEmpty method it will check for empty and the  null.